src/sentry/static/sentry/app/views/settings/incidentRules/presets.tsx

```diff
@@ -0,0 +1,303 @@
+  },
+  {
+    name: t('Users affected'),
+    match: /^count_unique\(tags\[sentry:user\]\)/,
+    validDataset: [Dataset.ERRORS],
+    default: 'count_unique(tags[sentry:user])',
+    /**
+     * Simple "Open in Discover" button
+     */
+    makeCTAParams: makeDefaultCTA,
+  },
+  {
+    name: t('Latency'),
+    match: /^(p[0-9]{2,3}|percentile\(transaction\.duration,[^)]+\))/,
+    validDataset: [Dataset.TRANSACTIONS],
+    default: 'percentile(transaction.duration, 0.95)',
+    /**
+     * see: makeGenericTransactionCta
+     */
+    makeCTAParams: opts =>
+      makeGenericTransactionCta({
+        opts,
+        tooltip: t('Latency by Transaction'),
+        tightPeriodScope: true,
+      }),
+  },
+  {
+    name: t('Apdex'),
+    match: /^apdex\([0-9.]+\)/,
+    validDataset: [Dataset.TRANSACTIONS],
+    default: 'apdex(300)',
+    /**
+     * see: makeGenericTransactionCta
+     */
+    makeCTAParams: opts =>
+      makeGenericTransactionCta({
+        opts,
+        tooltip: t('Apdex by Transaction'),
+        tightPeriodScope: true,
+      }),
+  },
+  {
+    name: t('Transaction Count'),
+    match: /^count\(\)/,
+    validDataset: [Dataset.TRANSACTIONS],
+    default: 'count()',
+    /**
+     * see: makeGenericTransactionCta
+     */
+    makeCTAParams: opts =>
+      makeGenericTransactionCta({
+        opts,
+        tooltip: t('Apdex by Transaction'),
+        tightPeriodScope: false,
+      }),
+  },
+  {
+    name: t('Error rate'),
+    match: /^error_rate\(\)/,
+    validDataset: [Dataset.TRANSACTIONS],
+    default: 'error_rate()',
+    /**
+     * "Open in discover" opens a query with same time range as the alert graph,
+     * columns [transaction.status, count()], and "Top 5 period" activated.
+     *
+     * We're guessing that a particular error type has gone up and it'll show
+     * in the breakdown
+     */
+    makeCTAParams: ({orgSlug, projects, incident, stats}) => ({
+      buttonText: t('Open in Discover'),
+      to: getIncidentDiscoverUrl({
+        orgSlug,
+        projects,
+        incident,
+        stats,
+        extraQueryParams: {
+          fields: ['transaction.status', 'count()'],
+          display: 'top5',
+        },
+      }),
+    }),
+  },
+];
+
+/**
+ * - CASE 1: If has a specific transaction filter
+ *   - CTA is: "View Transaction Summary"
+ *   - Tooltip is the transaction name
+ *   - the same period as the alert graph (i.e. with alert start time in the middle)
+ *
+ * - CASE 2: If txn is NOT filtered, or has a * filter:
+ *   - "Open in Discover" button with optional tooltip which opens a discover view with...
+ *      - req1: period [alert start time, alert end time] IF `tightPeriodScope`
+ *              is true, otherwise the period is the same as the alert graph.
+ *      - req2: fields {transaction, count(), <metric>} sorted by count()
+ *      - req3: showing <metric> on the y-axis, and top-5 activated
+ */
+function makeGenericTransactionCta(opts: {
+  opts: PresetCTAOpts;
```

Comment:
Can we be consistent with the acronym casing?

Reply:
Ah yeah, I've been getting mixed up on this recently, for some reason I thought we preferred real acronyms